buildenv/jenkins/omrbuild.groovy

```diff
@@ -223,7 +223,7 @@ def SPECS = [
         'junitPublish' : true
     ],
     'linux_x86-64_cmprssptrs' : [
-        'label' : 'Linux && x86',
+        'label' : 'compile:xlinux',
         'reference' : defaultReference,
```

Comment:
I'm not exactly sure what the label is used for, but is it correct that the x86 32-bit, 64-bit, and compressed refs build all have the same label?

We probably don't need a separate compressedrefs build anymore since it is on by default.  Maybe we should just get rid of it.

Reply:
`label` picks the machine. There are no 32bit machines. All are 64bit with 32bit libs. So imo having the same label for 32/64 linux is fine.
I can add a commit to remove the compressed version. I think `linux_x86-64` is used and cmp is dead code.

Reply:
There are some diffs between the specs though. @0xdaryl can you or someone confirm we no longer need `linux_x86-64_cmpssptrs`? It might be the cmp one wasn't using cmake? cc @rwy0717